target/linux/generic/pending-6.6/809-02-nvmem-layouts-add-support-for-ascii-env-driver.patch

```diff
@@ -106,6 +112,13 @@ Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
 +		info.offset = value - data;
 +		info.bytes = strlen(value);
 +		info.np = of_get_child_by_name(dev->of_node, info.name);
++		for_each_child_of_node(dev->of_node, child) {
++			if (of_property_read_string(child, "label", &label) &&
```

Comment:
`of_property_read_string()` can return negative on error (eg. if the property exists but type isn't string). you should check and handle that.

Reply:
Mhhh actually I think I should use !of_property_read_string